kubernetes/deployment.tmpl

```diff
@@ -1,21 +1,48 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: front-end-staging
+  name: front-end-project-staging
   labels:
-    app: front-end-staging
+    app: front-end-project-staging
 spec:
   replicas: 2
   selector:
     matchLabels:
-      app: front-end-staging
+      app: front-end-project-staging
   template:
     metadata:
       labels:
-        app: front-end-staging
+        app: front-end-project-staging
     spec:
       containers:
         - name: fe-project-staging
           image: zooniverse/fe-project:__IMAGE_TAG__
           ports:
             - containerPort: 3000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: front-end-project-production
```

Comment:
`front-end-project-production` doesn't really tell me which service this, could this be better named, e.g. `zooniverse-org-production`

Reply:
I agree. At the moment it's obvious that the "front end project" is the rebuild, but that won't make sense a couple of years from now.

Reply:
Okay - how does that fit in with specifying that this is the project app as opposed to the static pages / admin / whatever app, as these will be built out separately? Or will they share the same service?

Reply:
`zooniverse-org-project-production` vs `zooniverse-org-static-production` etc?

Reply:
Yeah - `zooniverse.org` will consist of multiple apps, which will have >1 environment (staging, production) etc, so would `zooniverse-org-project-production` for this one be more appropriate?